setup.py

```diff
@@ -64,8 +66,7 @@ def main():
         # for some reason the model/ directory had to be added separately
```

Comment:
and you can remove this comment as I don't think it is applicable if you are explicitly adding filenames instead of directory globs

Reply:
done